index.html

```diff
@@ -1218,7 +1218,9 @@ <h2>Core Vocabulary Definitions</h2>
 <code>unobserveproperty</code> or an  <a>Array</a> containing a combination of these terms.
     </span>
 </p>
-
+<p class="note">
+    It is considered to be good practice that each <code>observeproperty</code> has a corresponding <code>unobserveproperty</code> unless the protocol supports implicit unsubscription mechanisms (e.g., heartbeat to detect connection loss).
+</p>
```

Comment:
I'd also add this sentence: *Implementations might/should (?) use the "observerproperty" form as the default for "unobserverproperty" WoT interaction*. I think we can spare a little bit of redundancy here. For example:
```json
{
  "op": "observeproperty",
  "href": "mqtt://wot.example.com/level"
}
```
An implementation might infer that to *unobserveproperty* it needs to *unsubscribe* from `mqtt://wot.example.com/levelevent` topic.